toptal/picasso
# feat: allow 'defaultExpanded' prop for SidebarItem for plugging to re…

[FX-429](https://toptal-core.atlassian.net/browse/FX-429)

### Description

We want that Sidebar submenu is expanded by default when one of its children is `selected`

### How to test

1. click on different menu items in Expanded By Default
2. note that it re-renders with expanded sub menu when you navigate to one of the 2nd level items

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| ![before](https://user-images.githubusercontent.com/54671509/64314069-59da7f80-cfb6-11e9-91e6-9613aa01d9a6.gif) | ![after](https://user-images.githubusercontent.com/54671509/64314074-5f37ca00-cfb6-11e9-8104-3711d11d6a10.gif) |

### Review
